### PR TITLE
Send transactions back in block-info command

### PR DESCRIPTION
**Pull request by NullSoldier**

### CODE REVIEW

**Review by rohanjadvani — APPROVED** [review]

File: ironfish/src/rpc/routes/chain/getBlockInfo.ts [hunk 1]
```diff
@@ -42,19 +62,41 @@ router.register<typeof GetBlockInfoRequestSchema, GetBlockInfoResponse>(
...
     if (!header) {
       throw new ValidationError(`No block with hash ${request.data.hash}`)
     }
 
+    const block = await node.chain.getBlock(header)
+    if (!block) {
+      throw new ValidationError(`No block with hash ${request.data.hash}`)
+    }
+
+    const transactions: GetBlockInfoResponse['block']['transactions'] = []
```
Comment (brekk) [inline ironfish/src/rpc/routes/chain/getBlockInfo.ts:76]: For my own understanding, why is the type here in brackets? (I am assuming this is a TS thing I'm unfamiliar with.)

File: ironfish/src/rpc/routes/chain/getBlockInfo.ts [hunk 2]
```diff
@@ -42,19 +62,41 @@ router.register<typeof GetBlockInfoRequestSchema, GetBlockInfoResponse>(
...
+    })
+
     request.status(200).end({
       block: {
         graffiti: header.graffiti.toString('hex'),
         hash: request.data.hash,
         previousBlockHash: header.previousBlockHash.toString('hex'),
         sequence: Number(header.sequence),
         timestamp: header.timestamp.valueOf(),
+        transactions: transactions,
```
Comment (brekk) [inline ironfish/src/rpc/routes/chain/getBlockInfo.ts:99]: Again, might pitch the object property value shorthand here as well.
```suggestion
        transactions,
```

File: ironfish/src/rpc/routes/chain/getBlockInfo.ts (hunk 1 above)
Comment (NullSoldier) [reply ironfish/src/rpc/routes/chain/getBlockInfo.ts:76]: If you have a type, you can reference inner types using brackets in Typescript without redeclaring them.

```typescript
type Foo = {
    bar: {
        baz: number
    }
}

type Baz = Foo['bar']['baz']
// Type of baz is number
```

File: ironfish/src/rpc/routes/chain/getBlockInfo.ts (hunk 2 above)
Comment (NullSoldier) [reply ironfish/src/rpc/routes/chain/getBlockInfo.ts:99]: In reference to the other conversation here, https://github.com/iron-fish/ironfish/pull/336#discussion_r691532460

**Pull request merged**